conf/modules.config

```
@@ -105,6 +140,18 @@ process {
         ext.args = '--max-error 3 --adapter-length 250 --fwd-context ""'
     }
 
+    withName: VSEARCH_CLUSTER {
+        ext.args = "--minseqlength ${params.vsearch_minseqlength} --maxseqlength ${params.vsearch_maxseqlength} --qmask none --clusterout_sort --gapopen 0E/5I --gapext 0E/2I --mismatch -8 --match 6 --iddef 0 --minwordmatches 0 --qmask none -id ${params.vsearch_id}"
```

I am not sure why this is single dash for `id` prameter only `-id` ?

---

It is a typo, thanks for spotting! 
```suggestion
        ext.args = "--minseqlength ${params.vsearch_minseqlength} --maxseqlength ${params.vsearch_maxseqlength} --qmask none --clusterout_sort --gapopen 0E/5I --gapext 0E/2I --mismatch -8 --match 6 --iddef 0 --minwordmatches 0 --qmask none --id ${params.vsearch_id}"
```